src/lib.rs

```diff
@@ -101,11 +101,11 @@ pub mod rngs;
 pub mod seq;
 
 // Public exports
-#[cfg(all(feature = "std", feature = "std_rng"))]
+#[cfg(all(feature = "std", feature = "std_rng", feature = "getrandom"))]
 pub use crate::rngs::thread::thread_rng;
 pub use rng::{Fill, Rng};
 
-#[cfg(all(feature = "std", feature = "std_rng"))]
+#[cfg(all(feature = "std", feature = "std_rng", feature = "getrandom"))]
 use crate::distributions::{Distribution, Standard};
```

Comment:
This shouldn't depend on `getrandom`?

Reply:
These imports are only used for `random` which uses `thread_rng` which uses `OsRng` which uses `getrandom`.